lib/jekyll/related_posts.rb

```diff
@@ -42,7 +42,7 @@ def build_index
     end
 
     def lsi_related_posts
-      self.class.lsi.find_related(post.content, 11) - [post]
+      self.class.lsi.find_related(post, 11)
```

Comment:
[It shouldn't](https://github.com/jekyll/classifier-reborn/blob/133487722c311ff6c7f560cca40d393f36f70468/lib/classifier-reborn/lsi.rb#L47) -- both `add_item` and `lsi_related_posts` will call `#to_s` on what you pass into it. I'd assume you could also leave this function the same, and change `build_index` to call `add_item(x.content)`, but this way seemed clearer to me. 

Regardless, I feel like `build_index` and `lsi_related_posts` should be using the same kind of object -- either both Posts or both Strings. That way, you won't have to ` - [post]` at the end, because classifier-reborn is [already doing that on its side](https://github.com/jekyll/classifier-reborn/blob/133487722c311ff6c7f560cca40d393f36f70468/lib/classifier-reborn/lsi.rb#L237).